src/include/platform/ConnectivityManager.h

```diff
@@ -517,6 +537,16 @@ inline const char * ConnectivityManager::WiFiAPModeToStr(WiFiAPMode mode)
     return ImplClass::_WiFiAPModeToStr(mode);
 }
 
+inline const char * ConnectivityManager::WiFiStationStateToStr(WiFiStationState state)
```

Comment:
Since these a are common constants, could we represent  them as strings in the base class instead?

Following the current code, it returns NULL on noWifi for all states, however I would expect a state of "Connecting" to say "Connecting" rather than NULL. What is the benefit for splitting the text convesion into implementation classes?

Reply:
This is function to return constant string based on input state, if we represent it as a string, does this means we need to set its value for each state change? 

This function is same as the following existent APIs semantically. I prefer follow the existing pattern.
    static const char * WiFiStationModeToStr(WiFiStationMode mode);
    static const char * WiFiAPModeToStr(WiFiAPMode mode);

If a device does not has a WiFi chip, why return "Connecting" instead of NULL?